tutorial/simple-todos/01-creating-app.md

```diff
@@ -9,13 +9,10 @@ Install the latest official Meteor release [following the steps in our docs](htt
 
 ## 1.2: Create Meteor Project
 
-The easiest way to setup Meteor with Vue is by using the command `meteor create` with the option `--vue-2` and your project name:
-
-
-> Side note: Since meteorJS version 2.9 the default --vue will create a vue3 app, in order to follow this tutorial, use the vue2 command: ``--vue-2``
+The easiest way to setup Meteor with Vue is by using the command `meteor create` with the option `--vue` and your project name:
 
 ```
-meteor create --vue-2 simple-todos-vue --prototype
+meteor create --vue simple-todos-vue --prototype
```

Comment:
later on you ask the user to add autopublish package, you should either pass the `--prototype` or ask the user to add the autopublish package

Reply:
https://github.com/meteor/meteor/blob/cb6c97dc5f9897b61a5479d415a61ae2b61d1fac/tools/cli/commands.js#L1054-L1062

Reply:
I think we could change this
Maybe removing this `--prototype` and later on asking the user for the autopublish and other package that you have already mentioned

Reply:
Asking the user to add `autopublish` and `insecure` sounds good